src/api/macros/chrysalis-macros.js

```diff
@@ -0,0 +1,386 @@
+/* Chrysalis -- Kaleidoscope Command Center
```

Comment:
The flow from here to macros/index.js to App.js seems confusing to me. 
Is there a reason this file shouldn't just be src/api/macros.js?

Reply:
This could be `api/macros/index.js` itself. The reason it's not is because I originally copied another module that had a similar setup, and never bothered to change it.

I can certainly move it all to a single `src/api/macros/index.js`, or even `src/api/macros.js`. The former would be more consistent with the rest of the modules, some of which are similarly index.js + random other (focus, log, colormap are like that).

Reply:
I'm not a big fan of the index.js pattern, both because it's kind of nonsensical and because then all my editor tabs are for 'index.js'

Reply:
Hmm.... there's only 3 `index.js` files in `src/api/focus` currently, but... I suppose I can cut that down to 0.